### PR TITLE
New webpage

### PR DESCRIPTION
**Pull request by Laserlicht**
Here we can discuss about minimal requirements missing...

Current state: https://laserlicht.github.io/VCMI.eu/

### CODE REVIEW

**Comment by IvanSavenko** [comment]
- Not sure if there is any need to focus on Github. Majority of visitors here would be players, so easily noticeable links to Discord, Forum and maybe facebook would be more useful than developer-centered Github.

- Perhaps remove links that nobody bothers to maintain? ModDB and Steam have latest announcement for 1.1 and 1.2. Youtube has single 1-year old video.

- Some screenshots somewhere would be nice. For example reuse screenshots from Google Play.

Downloads page:
- Release page with perma-link to latest release ( https://github.com/vcmi/vcmi/releases/latest ) should be added as "other downloads" link below
- Windows release can be linked directly with similar permalink: https://github.com/vcmi/vcmi/releases/latest/download/VCMI-Windows.exe
- Possibly same for Android: Google Play, F-Droid, direct links for 2 apk's
- For other platforms, probably better to keep links to docs.

**Comment by DjWarmonger** [comment]
Why were all graphics deleted? They're hosted on VCMI.eu to be accessible from other places, in particular our main github page.

**Comment by DjWarmonger** [comment]
> Youtube has single 1-year old video.
Youtube has multiple playlists of videos published by others. Just because we don't record our own videos now doesn't mean it's not needed, or not helpful to others.

Anyway, link icons don't take much place.

**Comment by DjWarmonger** [comment]
Also, the main page background seems to be map taken straight from HoTA, not exactly fair or precise :P

I can provide some cool map screenshots with all the mods and features.

**Comment by IvanSavenko** [comment]
H3-only map might be more suitable. Or even plain Arrogance which many players will recognize. To reduce "buggy modders sandbox" associations.

**Comment by Laserlicht** [comment]
@IvanSavenko @DjWarmonger 

Improved mentioned points. I think it should be okay now. Any other easy points to fix?

I think it's ready now.

And I currently don't want to invest much more time in it (e.g. for improving mod repository).

**Comment by DjWarmonger** [comment]
Creature table contains almost no info:
image

All the stats should be there as a minimum.

**Comment by Laserlicht** [comment]
What info from Json should added?

**Comment by DjWarmonger** [comment]
Take a look at this table of Cove creaures, for example:
https://heroes.thelazy.net/index.php/Cove

**Comment by IvanSavenko** [comment]
Good to go, I guess we can publish it alongside with 1.5 release.

Minor tweaks for downloads screen:
- perhaps show platform icon before platform name? e.g. `[tux icon] Linux (Flathub)`
- for Android order should probably be Google Play, F-Droid, 64 bit, 32 bit
- "Needed files to play" -> perhaps "Required flies to play"?

Oh, and old page also had:
- "Hosting provided by Digital Ocean" section
- And that "VCMI, this page and the VCMI Team are in no way associated with or supported by Ubisoft Entertainment..." paragraph.

Probably should be added to footer in new website

File: mkdocs.yml [hunk 1]
```diff
@@ -63,7 +63,11 @@ extra_css:
 extra_javascript:
   - https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js
   - theme/tablesort.js
-copyright: Copyright &copy; 2007 - <span id="copyyear"></span><script>document.getElementById('copyyear').innerHTML = new Date().getFullYear();</script> VCMI team<br>VCMI is not affiliated with Ubisoft. Buy Heroes III on <a href="https://www.gog.com/en/game/heroes_of_might_and_magic_3_complete_edition">gog.com</a>, we don't support piracy.
+copyright: >
+  Copyright &copy; 2007 - <span id="copyyear"></span><script>document.getElementById('copyyear').innerHTML = new Date().getFullYear();</script> VCMI team
+  <p style="font-size:.42rem !important; line-height: 0.5 !important;">VCMI, this page and the VCMI Team are in no way associated with or supported by Ubisoft Entertainment...</p>
```
Comment (IvanSavenko) [inline mkdocs.yml:68]: Sorry, I meant full version, as it is on our current website:
```suggestion
  <p style="font-size:.42rem !important; line-height: 0.5 !important;">VCMI, this page and the VCMI Team are in no way associated with or supported by Ubisoft Entertainment. Heroes, Might and Magic, Heroes of Might and Magic and their respective logos are registered trademarks of Ubisoft Entertainment in the U.S. and/or other countries. All Rights Reserved. All other trademarks are properties of their respective owners.</p>
```

**Comment by DjWarmonger** [comment]
Much better now!

Just a remark: fightValue can be removed, it's not used remnant for OH3 files. I'm going to deprecate it after 1.5.

Only aiValue is neccessary.

**Comment by IvanSavenko** [comment]
FAQ probably need an update to account for 1.5 state of affairs.
Or even moved to vcmi/vcmi repo and imported like rest of docs.

Can't comment/suggest on outdated lines for some reasons, so here, two points that need changes. Everything else seems to be up to date.

* When will it be possible to play online in multiplayer?
> As of VCMI 1.5.0, multiplayer has reached a reasonable level of stability and is playable, including support for simultaneous turns and turn timers. There is a lobby to play online with anyone in the world, which can be accessed from within the game: New Game -> Multiplayer -> Online Service (second button from the top). Alternatively, you can play over a local network using any supported device. Further development of multiplayer is definitely on our priority list.

* What are the options to customize AI?
> As of VCMI 1.5.0, the default adventure map AI algorithm is Nullkiller AI. It should be fairly challenging, however it may work slowly on low-end devices. There is an alternative faster but simpler VCAI, which you can choose in VCMI launcher settings. You may also check the Resourceful AI mod, which provides options to boost AI and to speed it up.

(also, is 'Resourceful AI' mod name actual? I think there were some changes with this mod?)

File: src/build_mod_page.py [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+                image = ""
+                if "iconLarge" in mod["config"]["creatures"][k]["graphics"]:
+                    image = Html.image(path=modparser.get_image_base64(mod, mod["config"]["creatures"][k]["graphics"]["iconLarge"]), size='50')
+                mdModTable.extend([
+                    mod["config"]["creatures"][k]["name"]["singular"],
+                    image,
+                    get_value_if_exists(mod["config"]["creatures"][k], "level"),
+                    get_value_if_exists(mod["config"]["creatures"][k], "attack"),
+                    get_value_if_exists(mod["config"]["creatures"][k], "defense"),
+                    str(mod["config"]["creatures"][k]["damage"]["min"]) + "-" + str(mod["config"]["creatures"][k]["damage"]["max"]) if "damage" in mod["config"]["creatures"][k] else "",
```
Comment (DjWarmonger) [inline src/build_mod_page.py:56]: Add spaces around "-"

**Pull request merged**